hack/tools/tilt-prepare/main.go

```diff
@@ -283,9 +287,11 @@ func tiltResources(ctx context.Context, ts *tiltSettings) error {
 	// NOTE: strictly speaking cert-manager is not a resource, however it is a dependency for most of the actual resources
 	// and running this is the same task group of the kustomize/provider tasks gives the maximum benefits in terms of reducing the total elapsed time.
 	if ts.DeployCertManager == nil || *ts.DeployCertManager {
-		tasks["cert-manager-cainjector"] = preLoadImageTask(fmt.Sprintf("quay.io/jetstack/cert-manager-cainjector:%s", config.CertManagerDefaultVersion))
-		tasks["cert-manager-webhook"] = preLoadImageTask(fmt.Sprintf("quay.io/jetstack/cert-manager-webhook:%s", config.CertManagerDefaultVersion))
-		tasks["cert-manager-controller"] = preLoadImageTask(fmt.Sprintf("quay.io/jetstack/cert-manager-controller:%s", config.CertManagerDefaultVersion))
+		if ts.PreloadCertManagerImages == nil || *ts.PreloadCertManagerImages {
```

Comment:
Fine with the change in general - but this 'defaulting to true` setup seems kind of weird. Could we clean this up by having a defaulting function for tiltSettings?

Reply:
We have the defaulting function (I also adjusted it). This is just double-checking that we don't panic. The same we have for DeployCertManager.

I would be fine with dropping the nil check in both cases

Reply:
Yeah - it's just slightly confusing. I guess the alternative would be to default to a struct with no pointers, but it's immaterial for this PR.

Reply:
Got rid of the new flag :D